pkg/ibmvpc/vpc.go

```diff
@@ -125,6 +134,14 @@ type VPC struct {
 	vpcmodel.VPCResource
 	nodes             []vpcmodel.Node
 	connectivityRules *vpcmodel.ConnectivityResult // allowed connectivity between elements within the vpc
+	zones             map[string]*Zone
+}
+
+func (v *VPC) getZoneByName(name string) (*Zone, error) {
+	if z, ok := v.zones[name]; ok {
+		return z, nil
+	}
+	return nil, errors.New("zone not found")
```

Comment:
Might be helpful to add zone name and vpc name to the error message

Reply:
done